bin/asl-setup-dkms-mok

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/bash
+#
+
+if [ "${USER}" != "root" ]; then
+	echo "Must be run as root"
+	exit 1
+fi
+
+if [ ! -x /usr/bin/mokutil ]; then
+	echo "Missing /usr/bin/mokutil"
+	exit 1
+fi
+
+SBSTATE=$(mokutil --sb-state | awk '{print $2}')
+if [ "${SBSTATE}" != "enabled" ]; then
+	echo "This system is not running SecureBoot"
+	exit 0
+fi
+
+echo "!!!"
```

Comment:
FYI `echo` will take quoted multiline strings. For instance
```
echo "
This is a
multiline
statement to print out
"
```
Up to you which style is better, but it might save the hassle of managing all the double quotes on each line.